code/drasil-printers/Language/Drasil/HTML/Print.hs

```diff
@@ -225,6 +225,7 @@ pOps Point    = "."
 pOps Perc     = "%"
 pOps LArrow   = " &larr; "
 pOps RArrow   = " &rarr; "
+pOps ForAll   = " ForAll "
```

Comment:
By any chance, did you try [`&forall;`](https://www.toptal.com/designers/htmlarrows/math/for-all/) here to see if that works?

Reply:
hmm i'll do that now